src/components/Employee/Compensation/Edit.tsx

```diff
@@ -17,14 +25,21 @@ export const Edit = () => {
   const format = useNumberFormatter('currency')
   const Components = useComponentContext()
 
+  const stateWcRiskOptions = useMemo(
+    () => WA_RISK_CLASS_CODES.map(({ code, description }) => ({ value: code, label: description })),
+    [],
+  )
```

Comment:
Wonder if it's worth it to break this into a dedicated component and lazy load it similar to what we are doing in company industry? probably overkill if we plan on getting the api work done soon. But if not, would be nice to avoid any performance hits just for the washington case